pkg/project/testoutput/openshift/ci-operator/build-image/Dockerfile

```diff
@@ -1,16 +1,13 @@
 # DO NOT EDIT! Generated Dockerfile.
```

Comment:
I tried this Dockerfile and got:
```
docker run -it --rm localhost/kubectl-oc:latest oc
oc: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by oc)
oc: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by oc)
oc: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by oc)
```

Reply:
Then again 4.15 would not be in sync with cluster version, although it would not cause trouble most likely.

Reply:
Maybe we could download oc from somewhere else? OCP cluster serves it at some HTTP endpoint?

Reply:
4.15 is based on RHEL8 too.

Reply:
or download it from https://access.redhat.com

Reply:
So seems not the OCP alignment with the builder image is important, but the RHEL version

Reply:
yep, more specifically glibc version

Reply:
Still it would be nice to hat ocp version also aligned.

Reply:
builder.glibc >= tools.glibc must hold

Reply:
> OCP cluster serves it at some HTTP endpoint?

I am afraid that `oc` would also use too new glibc.

Reply:
@dsimansk give us a hint:
https://github.com/openshift-knative/kn-plugin-event/blob/main/openshift/ci-operator/build-image/Dockerfile#L8